bin/release

```diff
@@ -466,7 +466,7 @@ done
 
 # Create the tar file
 test "$verbose" && echo "   Running tar..." 1>&2
-(cd "$tmpdir" && exec tar -ch --exclude-vcs -f "$HDF5_VERS.tar" "./$HDF5_IN_VERS" || exit 1 )
```

Comment:
I would have used some variation of --exclude=\.*, but everything I tried either didn't exclude the files beginning with '.', or excluded all files.  Multiple --exclude options for each file works, but one to cover all would be preferred.